modules/core/src/index.js

```diff
@@ -88,6 +89,7 @@ export {AmbientLight, PointLight, DirectionalLight} from '@luma.gl/core';
 // Exports for layers
 // Experimental Features may change in minor version bumps, use at your own risk)
 export const experimental = {
+  Sunlight,
```

Comment:
This cannot be tree shaken. Our current convention for experimental classes is with underscore name.